app/src/main/java/com/android/sample/ui/utils/SearchBar.kt

```diff
@@ -31,7 +32,11 @@ import com.android.sample.resources.C.Tag.SEARCH_BAR_CORNER_RADIUS
  */
 @Preview
 @Composable
-fun SearchBar(modifier: Modifier = Modifier, onValueChange: (String) -> Unit = {}) {
+fun SearchBar(
+    modifier: Modifier = Modifier,
+    onValueChange: (String) -> Unit = {},
+    isSingleLine: Boolean = false
```

Comment:
Is the parameter isSingleLine necessary ? Can't you put it directly to true in the composable TextField ? 

In addition, why is it by default to false ? For a search bar, we don't want to write multiple lines, don't we?